src/generic/FormSwitchGroup.jsx

```diff
@@ -37,9 +37,11 @@ export default function FormSwitchGroup({
           />
         </div>
         <Form.Text
-          className="mt-0 pr-3"
+          className="mt-0 pr-3 text-gray-700"
+          style={{
+            fontSize: '1.125rem',
```

Comment:
body text size is already 18 px which is equal to 1.125 rem. why we want to pass this size in styles. even if we need to pass font size can we make use of some class paragon? check this for details
https://paragon-edx.netlify.app/foundations/typography

Reply:
Yes, the **body text size** is `18px` but there is CSS applied to the `Form.text` that specifies `0.875rem` (`14px`) which overrides the body text size. Hence, this style is added to override that CSS applied on top of the **body text size** to `Form.text`.

Reply:
Okay got it